.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Default owners for everything in the repo unless a later match takes precedence
+*       @sonatype/nxrm-techleads
```

Comment:
weird spacing, also a missing newline at the end of the file

Reply:
Why the newline? Github's sample files do not contain a newline at the end of the file.

Reply:
We put newlines at the end of all our files as a matter of style, it probably doesn't really matter

Reply:
presumably we use that as a style because of https://stackoverflow.com/a/729795

Reply:
Thanks for the explanation. Fixed